test/cases/fixes/noInferableTypes/variableDeclarations/expected.ts

```diff
@@ -65,22 +65,41 @@
 	let letTakesChild = new Child();
 
 	// Non-inferable const direct class instances
```

Comment:
```suggestion
	// Inferable const direct class instances
```

...honestly, if you want to just delete all these existing comments, I'd be up for that too. In the years since writing I've grown much less fond of comments that basically restate the code. And these are IMO a symptom of the test being ripe for splitting up, per #1504.